packages/fuel-gauge/src/coverage-contract.test.ts

```diff
@@ -523,12 +512,12 @@ describe('Coverage Contract', () => {
       .echo_struct_vector_last([
         {
           foo: 1,
-          bar: 11337n,
+          bar: 11337,
           baz: '123456789',
         },
         {
           foo: 2,
-          bar: 21337n,
+          bar: 21337,
```

Comment:
```suggestion
          bar: 21337n,
```

Why are we removing the BigInt expression here?

Reply:
https://github.com/FuelLabs/fuels-ts/pull/2789#discussion_r1682778529